### ISSUE TITLE
No ideal results

### ISSUE THREAD

**Issue by wzx95789** [FingerRec/Self-Supervised-Temporal-Discriminative-Representation-Learning-for-Video-Action-Recognition#5]
Hi,thanks for your good work! I'm very interested in your work and follow. I run the self-supervised learning to achieve the pretrained weight, and in the step 2 transfer to action recognition, i get 47, the best result on HMDB51,and in the test, the result gets less than 1%  improvement. I also use your provided pretrained weight to transfer to action recognition, but do not get the ideal results.  I checked my procedure and try to increase the epoch, but it seems no use. The only difference is that in the step 1, i changed the batchsize from 16 into 8. I do not know if i miss some information or steps, could you give me some advice or help?

**Comment by FingerRec** [comment]
Hi wzx,
Thanks for your interests in our work and good question!


There exist some points need to be noticed:
(1). Larger batch size lead to better result and have large influence in final result. As we find V100 GPU lead to 2% better than 1080Ti when use all memory.
(2). To improve reproducibility , I offer pretrained weight that trained only use one single 1080Ti GPU, directly run given code in low-compute ability machine like 1080Ti should also achieve at least 49+ clip top-1 accuracy (I test it just now).
(3). To get result large than 50 is really easy. Add the memory bank size and carefully select the TCA hyperparameter in pretrain should lead to better result.
(4). Test models include time slide windowing and corner cropping for best result, corner cropping is not include in test.py for time consume.
